code/utils.q

```diff
@@ -17,4 +17,4 @@ i.loadNLP:{
 // @fileoverview Used throughout the library to convert linux/mac file names to windows equivalent
 // @param path {str} the linux 'like' path
 // @retutn {str} path modified to be suitable for windows systems
-i.ssrwin:{[path]$[.z.o like "w*";ssr[path;"/";"\\"];path]}
```

Comment:
You need to change the ssrwin function used in `code/nodes/featureCreation/nlp/funcs.q` too